modules/local/bedtools/main.nf

```diff
@@ -0,0 +1,39 @@
+process INTERSECT {
+    label "process_low"
+    label "stage"
+    label "scratch"
+    tag "${meta.id}"
+
+    input:
+        tuple val(group), val(meta), file(vcf)
+        val(intersect)
+        
+    output:
+        tuple val(group), val(meta), file("${prefix}.intersected.${suffix}"), emit: intersected
+        path "versions.yml",                                      emit: versions
+
+    script:
+        def args = task.ext.args ?: ''
+        prefix = task.ext.prefix ?: "${meta.id}.${meta.type}"
+        suffix = task.ext.suffix ?: ''
```

Comment:
It's always good to provide a default value in the script for prefix, suffix. Later they can be customized via the config.

```
suffix = task.ext.suffix ?: "bed"
```